pkg/cmd/cli/cmd/newapp.go

```diff
@@ -612,6 +612,14 @@ func handleRunError(err error, baseName, commandName, commandPath string) error
 		transformError(err, baseName, commandName, commandPath, groups)
 	}
 	buf := &bytes.Buffer{}
+	if len(config.ArgumentClassificationErrors) > 0 {
+		fmt.Fprintf(buf, "Errors occurred while determining argument types:\n")
+		for _, classErr := range config.ArgumentClassificationErrors {
+			fmt.Fprintf(buf, fmt.Sprintf("\n%s:  %v\n", classErr.Key, classErr.Value))
+		}
+		fmt.Fprint(buf, "\n")
+	}
+	fmt.Fprint(buf, "Errors occurred during resource creation:\n")
```

Comment:
this should also be conditional on there actually being at least one resource creation error.